src/main/java/core/org/akaza/openclinica/bean/odmbeans/DiscrepancyNoteBean.java

```diff
@@ -82,4 +83,12 @@ public String getThreadUuid() {
     public void setThreadUuid(String threadUuid) {
         this.threadUuid = threadUuid;
     }
+
+    public String getDisplayId() {
```

Comment:
not sure how these get displayed in the ODM, but we want to use the existing "ID" attribute and not add a new attribute called "DisplayID". Only the database will have this new column and the "ID" attribute value will be populated from this column.

Reply:
I made changes to display it as "ID" attribute